stock_inventory_valuation_pivot/views/stock_account_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_inventory_valuation_search" model="ir.ui.view">
+        <field name="model">stock.valuation.layer</field>
+        <field name="inherit_id" ref="stock_account.view_inventory_valuation_search" />
+        <field name="arch" type="xml">
+            <field name="product_tmpl_id" position="after">
+                <field
+                    string="Date Before"
+                    name="create_date"
+                    filter_domain="[('create_date', '&lt;=', self)]"
+                />
+            </field>
+        </field>
+    </record>
+
+    <record id="stock_valuation_layer_view_pivot" model="ir.ui.view">
+        <field name="name">stock.valuation.layer.pivot</field>
+        <field name="model">stock.valuation.layer</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="product_id" interval="month" type="row" />
```

Comment:
the field product_id is not a date field so you can not do that... change the this field by other of date type..

![Selección_686](https://user-images.githubusercontent.com/7689807/132258273-5b837d07-7aa2-4241-97b9-e3e7ed4687a1.png)

Reply:
well, the intention was to set product_id and not set `interval="month"`, so I'm going to remove `interval="month"`